src/Compilers/CSharp/Test/Semantic/Semantics/RefEscapingTests.cs

```diff
@@ -730,14 +762,19 @@ static void Main()
     }
 }
 ";
-            CreateCompilationWithMscorlibAndSpan(text).VerifyDiagnostics(
+            CreateCompilationWithMscorlibAndSpan(text, parseOptions: TestOptions.Regular10).VerifyDiagnostics(
                 // (12,36): error CS8156: An expression cannot be used in this context because it may not be returned by reference
                 //         return ref ReturnsRef1(out var _);
                 Diagnostic(ErrorCode.ERR_RefReturnLvalueExpected, "var _").WithLocation(12, 36),
                 // (12,20): error CS8347: Cannot use a result of 'Program.ReturnsRef1(out int)' in this context because it may expose variables referenced by parameter 'x' outside of their declaration scope
                 //         return ref ReturnsRef1(out var _);
                 Diagnostic(ErrorCode.ERR_EscapeCall, "ReturnsRef1(out var _)").WithArguments("Program.ReturnsRef1(out int)", "x").WithLocation(12, 20)
                 );
+            CreateCompilationWithMscorlibAndSpan(text).VerifyDiagnostics(
+                // (18,20): error CS8166: Cannot return a parameter by reference 'x' because it is not a ref or out parameter
+                //         return ref x;
+                Diagnostic(ErrorCode.ERR_RefReturnParameter, "x").WithArguments("x").WithLocation(18, 20)
```

Comment:
Yes, I have several PROTOTYPE comments for `ERR_RefReturnParameter` when it is reported for an `out` parameter. I agree, it seems reasonable to report "because it is not a ref parameter" only.